zstacklib/zstacklib/utils/gpu.py

```
@@ -233,3 +234,55 @@ def is_hygon_gpu_cmd(pci_addr):
+        if output:
+            yield output.strip()
+
+
+def extract_number(s):
+    if s is None or s == "":
+        return None
+
+    if isinstance(s, (int, float)):
+        return float(s)
+
+    match = re.search(r'(\d+(?:\.\d+)?)\s*MB/S', s, flags=re.IGNORECASE)
```

Ruff (0.8.2)

`re` may be undefined, or defined from star imports

(F405)